### ISSUE TITLE
Offer ability to display translations on page of orphan sentences

### ISSUE THREAD

**Issue by alanfgh** [Tatoeba/tatoeba2#1943]
The list of orphan sentences (for example, https://tatoeba.org/eng/Activities/adopt_sentences/eng ) does not display translations. This is inconsistent with other list pages, which offer the choice. The fact that translations are not visible without going to the page for each sentence means that one cannot easily see which sentences already have translations and thus might require special care when it comes to modification.

**Comment by trang** [comment]
I'm not sure if displaying translations would be the best solution. Instead, we may want to remove the possibility to edit from the adopt page. 

If a sentence requires editing, it would be better if the user would first go to the sentence's page in case there are comments on the sentence. They can then edit the sentence from the sentence's page after checking for comments and translations.

I'm also concerned that by displaying the translations, users who are not yet very familiar with Tatoeba have the impression that they are somehow adopting the whole group of sentence + translations.

**Comment by alanfgh** [comment]
Then you'd have to do a lot of clicking on sentences to go back and forth. I guess it would be manageable, but it would be more work.

There are already a lot of clues that you're adopting sentences, not their translations:

(1) The title: "Orphan sentences in English". It does not say "Orphan sentences in English and their translations".

(2) The "About adoption" panel on the right says:

"Adopting is a way to vote 'this sentence is correct'. It is also an occasion to check the sentence and correct it if there is a mistake."

It says nothing about translations.

(3) Under "Tips", the page says "If you see another username appear after adopting a sentence, it means that someone else adopted the sentence very shortly before you did."

Again, this says nothing about translations.

But if you think that people might think that newcomers might assume that they're adopting translations, there are two things you can do:

(1) Add a note to the first panel explaining that you're only adopting the sentences, not the translations.

(2) Perform tests on novice users to see whether they interpret the page correctly.

**Comment by trang** [comment]
If more than 1 sentence out of 10 needs to be edited, then I can imagine it would indeed be cumbersome to have to go to the sentence's page to edit.

But I'm wondering, assuming that people follow the guideline of not changing the meaning of the sentence when editing it, is there still a necessity to show translations? As long as the contributor strives to fix the sentence without changing its meaning, I think there is no need for them to know what are the translations.

Should we instead simply add a few lines about not changing the meaning of the sentence when editing it?

**Comment by trang** [comment]
On a side note, I don't think it would be harmful to add a dropdown "Show translations", similar to the "Browse by language" page or the list page, with the default being "None". That would be an okay solution if we consider that it happens often enough that a contributors needs to see the translations on the adopt page.

**Comment by alanfgh** [comment]
> But I'm wondering, assuming that people follow the guideline of not changing the meaning of the sentence when editing it, is there still a necessity to show translations?

Yes, because sometimes the meaning is not clear from the sentence itself, and sometimes the wording is so messed up that it's hard to figure out how to improve it. If there is no translation, I will sometimes delete the sentence. If there is a translation, I will read it, or post a question to someone else who might understand it, or possibly use Google Translate on it, in order to help figure out what was intended by the original sentence.

**Comment by Poulpisator** [comment]
Having correcting a lot of orphan French sentences, I partially agree with alan. Translations are an important tool to understand the meaning of the sentence. However don't you think that the most important is actually the log? 
In my case, I usually check the log, and if the French sentence was the original, I correct it as I judge correct and leave messages / tags on the translations. 
If the French sentence was a translation of a sentence, I try to correct it to fit this translation and leave messages / tags on the translations.
(If it's untranslated, like you, I sometimes delete it)
